### PR TITLE
[blog] Adding a new community update blogpost.

### PR DESCRIPTION
**Pull request by morsapaes**
Community update for March'20 covering the most recent milestones and developments in the Flink community.

### CODE REVIEW

File: _posts/2020-03-30-community-update.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+In a [recent update](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/DISCUSS-Update-on-Flink-Stateful-Functions-what-are-the-next-steps-tp38646.html), some new features were announced, like multi-language support (including a Python SDK), function unit testing and Stateful Functions’ own flavor of the [State Processor API](https://ci.apache.org/projects/flink/flink-docs-stable/dev/libs/state_processor_api.html). The release cycle will be independent from core Flink releases and the Release Candidate (RC) has been created — so, **you can expect Stateful Functions 2.0 to be released very soon!**
+
+## Building up to Flink 1.11
+
+Amidst the usual outpour of discussion threads, JIRA tickets and FLIPs, the community is working full steam on bringing Flink 1.11 to life in the next few months. The upcoming release will focus on new features and integrations that broaden the scope of Flink use cases, as well as core runtime enhancements to streamline the operations of complex deployments.
```
Comment (rmetzger) [inline _posts/2020-03-30-community-update.md]:
```suggestion
Amidst the usual outpour of discussion threads, JIRA tickets and FLIPs, the community is working full steam on bringing Flink 1.11 to life in the next few months. The feature freeze is currently scheduled for late April, so the release is expected around mid May. 
The upcoming release will focus on new features and integrations that broaden the scope of Flink use cases, as well as core runtime enhancements to streamline the operations of complex deployments.
```

File: _posts/2020-03-30-community-update.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+There is a clear upward trend in the number of contributions to the repository, based on the number of commits. This reflects the **fast pace of development** the project is experiencing and also the **successful integration of the China-based Flink contributors** started early last year. To complement these observations, the repository registered a **1.5x increase in the number of individual contributors in 2019**, compared to the previous year. 
+
+But did this increase in capacity produce any other measurable benefits?
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/2020-03-30-flink-community-update/2020-03-30-flink-community-update_2.png" width="725px" alt="GitHub 2"/>
+</center>
+
+If we look at the average time of Pull Request (PR) “resolution”, it seems like it did: **the average time it takes to close a PR these days has been steadily decreasing** since last year, sitting between 5-6 days for the past few months.
```
Comment (rmetzger) [inline _posts/2020-03-30-community-update.md]: The graphs seem a bit inconsistent?
Why are 2018 and 2019 shown as bars, and 2020 is shown as a line? (in the contributors chart)
For the PR chart, the roles assignment of lines and bars has changed again.
Why can't use use bars for all years? Or at least use the line always for the current year?

**Comment by rmetzger** [comment]
Thanks a lot! It's a good read. Except for two comments (both are just suggestions), I'm +1 on publishing this.

File: _posts/2020-03-30-community-update.md (hunk 2 above)
Comment (morsapaes) [reply _posts/2020-03-30-community-update.md]: The idea was to make the chart as easy to interpret as possible, which was not the case with the formatting you mention (I tried a couple of different customisations and charts). For the PR data, for example, it's more valuable to see the "worst performance" isolated to contrast with where we are today than to have today's values lost in the lower end of the other bars.

Screen Shot 2020-03-31 at 10 28 50

Let me know if this makes sense to you.

Comment (rmetzger) [reply _posts/2020-03-30-community-update.md]: Yes, this is better, thanks :)

File: _posts/2020-03-30-community-update.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+	
+## New Committers and PMC Members
+
+The Apache Flink community has welcomed **1 PMC (Project Management Committee) Member** and **5 new Committers** since the last update (September 2019):
+
+### New PMC Members
+	Jark Wu
+
+### New Committers
+	Zili Chen, Jingsong Lee, Yu Li, Dian Fu, Zhu Zhu
```
Comment (fhueske) [inline _posts/2020-03-30-community-update.md]: plus Konstantin Knauf

File: _posts/2020-03-30-community-update.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+Throw into the mix improvements around type systems, the WebUI, metrics reporting and supported formats, this release is bound to keep the community busy. For a complete overview of the ongoing development, check [this discussion](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/DISCUSS-Features-of-Apache-Flink-1-11-td38724.html#a38793) and follow the weekly updates on the Flink [@community mailing list](https://flink.apache.org/community.html#mailing-lists).
+	
+## New Committers and PMC Members
+
+The Apache Flink community has welcomed **1 PMC (Project Management Committee) Member** and **5 new Committers** since the last update (September 2019):
+
+### New PMC Members
+	Jark Wu
```
Comment (fhueske) [inline _posts/2020-03-30-community-update.md]: plus Zhijiang Wang and Dawid Wysakowicz

File: _posts/2020-03-30-community-update.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+### New Committers
+	Zili Chen, Jingsong Lee, Yu Li, Dian Fu, Zhu Zhu
+
+Congratulations to all and thank you for your hardworking commitment to Flink!
+
+# The Bigger Picture
+
+## A Look into the Flink Repository
+
+In the [last update](https://flink.apache.org/news/2019/09/10/community-update.html), we showed off some numbers around Flink releases and mailing list activity. This time, we’re looking into the activity in the Flink repository and how it’s evolving.
```
Comment (fhueske) [inline _posts/2020-03-30-community-update.md]: "...showed off with some numbers..." or "showed some numbers"?

**Review by fhueske — COMMENTED** [review]
Thanks for the post @morsapaes!
I added a few comments :-)

Cheers, Fabian

File: _posts/2020-03-30-community-update.md (hunk 3 above)
Comment (morsapaes) [reply _posts/2020-03-30-community-update.md]: Thought he wasn't announced yet. 👀 Will gladly add!

File: _posts/2020-03-30-community-update.md (hunk 4 above)
Comment (rmetzger) [reply _posts/2020-03-30-community-update.md]: But they haven not been announced yet, so nobody knows

Comment (morsapaes) [reply _posts/2020-03-30-community-update.md]: Should I include them or...?

Comment (rmetzger) [reply _posts/2020-03-30-community-update.md]: The normal flow of things is that this gets announced on the ML (not in blog posts), thats why I would not include them

Comment (fhueske) [reply _posts/2020-03-30-community-update.md]: At least Dawid and Zhijiang are listed here: http://people.apache.org/phonebook.html?pmc=flink

Comment (morsapaes) [reply _posts/2020-03-30-community-update.md]: I agree with Robert, so didn't include them. Will do so in the next update!

**Review by rmetzger — APPROVED** [review]

**Comment by rmetzger** [comment]
Thanks a lot for this blog post! Merging now

**Pull request closed without merging**